database/database.go

```diff
@@ -45,6 +54,31 @@ func New(dialect, connection, defaultUser, defaultPass string, strength int, cre
 	return &GormDatabase{DB: db}, nil
 }
 
+func prepareBlobColumn(dialect string, db *gorm.DB) error {
+	alterBlobSQLStmtTpl := "alter table `%s` modify column %s %s"
```

Comment:
what does tpl mean? Would it be possible to use modify column https://gorm.io/docs/migration.html#ModifyColumn it would look better:
```
db.Model(&model.Message{}).ModifyColumn("extras", blobType)
```

Reply:
Template. Okay I will use `ModifyColumn`.